### PR TITLE
Hotfix for ray psutil on macOS

### PR DESCRIPTION
**Pull request by qiyuangong**
#2832

### CODE REVIEW

File: pyzoo/zoo/ray/raycontext.py [hunk 1]
```diff
@@ -63,6 +63,9 @@ def kill_redundant_log_monitors(redis_address):
     import subprocess
     log_monitor_processes = []
     for proc in psutil.process_iter(["name", "cmdline"]):
+        # Avoid throw exception when listing lwsslauncher in macOS
+        if proc.name == "lwsslauncher":
```
Comment (yangw1234) [inline pyzoo/zoo/ray/raycontext.py:67]: does filter status == ‘zombie’ work？

Comment (qiyuangong) [reply pyzoo/zoo/ray/raycontext.py]: maybe. But sometimes the process you are looking for maybe a zombie process

**Comment by qiyuangong** [comment]
http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-UnitTests-2.4-PYTHON-MAC/1/console passed, thk  to @pinggao18

**Comment by qiyuangong** [comment]
Jenkins passed http://10.239.47.210:18888/job/ZOO-PR-Validation/4204/

File: pyzoo/zoo/ray/raycontext.py [hunk 2]
```diff
@@ -63,11 +63,22 @@ def kill_redundant_log_monitors(redis_address):
...
+            is_same_redis = "--redis-address={}".format(redis_address)
+            if is_log_monitor and is_same_redis in cmdline:
+                log_monitor_processes.append(proc)
+        except psutil.AccessDenied:
+            # psutil may encounter AccessDenied exceptions
+            # when it's trying to visit core services
+            if psutil.MACOS:
+                continue
+            else:
+                raise
```
Comment (hkvision) [inline pyzoo/zoo/ray/raycontext.py:81]: raise what?

**Pull request merged**